packages/fewer/src/example.ts

```diff
@@ -1,14 +1,29 @@
 import { createVirtuals } from '@fewer/virtuals';
+import Schema from './Schema';
 import { createRepository } from './Repository';
 
-interface IUser {
-  firstName: string;
-  lastName: string;
-  deleted: boolean;
-}
-
-const Users = createRepository<IUser>('users');
-export const DeletedUsers = Users.where({ deleted: [true, true] }).where({deleted: false});
+const schema = new Schema(20080906171750)
+  .createTable('users', { force: true }, t => ({
+    firstName: t.optional(t.string),
+    lastName: t.string,
+    deleted: t.boolean,
+    createdAt: t.datetime,
+    updatedAt: t.datetime,
+  }))
+  .createTable('products', { force: true }, t => ({
```

Comment:
This style of immutable chaining seems to be a common pattern. I could also see people getting confused by it, so we should probably call it out in documentation.